meditation_app/dom.js

```diff
@@ -3,51 +3,66 @@ const app = () => {
     const play = document.querySelector('.play');
     const outline = document.querySelector('.moving-outline circle');
     const video = document.getElementById('video');
-    const replay = document.querySelector('.replay');
 
-    // Types of sounds
-    const musicTypes = document.querySelectorAll('.misic-type');
     // Timer
-    const timer = document.querySelector('.timer');
+    const minutesArea = document.querySelector('.minutes');
+    const secondsArea = document.querySelector('.seconds')
+
     // Moving-outline length
     const outlineLength = outline.getTotalLength();
-    // Time selector
-    const timeSelectors = document.querySelectorAll('.time-selector__item');
+
     // Time duration
-    let maxDuration = 600;
+    let defaultDuration = 600;
+
+    // Music repetition counter
+    let counter = 0;
 
     outline.style.strokeDasharray = outlineLength;
     outline.style.strokeDashoffset = outlineLength;
 
-    // Change music
-    musicTypes.forEach(musicType => {
-        musicType.addEventListener('click', () => {
-            music.src = musicType.getAttribute("data-sound");
-            video.src = musicType.getAttribute("data-video");
-            checkPlaying(music);
-        });
-    });
-
-    // Play music
-    play.addEventListener('click', () => {
+    function changeMusicType(id) {
+        let typeButton = document.getElementById(id).parentElement;
+        music.src = typeButton.getAttribute("data-sound");
+        video.src = typeButton.getAttribute("data-video");
         checkPlaying(music);
-    });
+        console.log(typeButton);
+    };
 
-    // Replay music
-    replay.addEventListener('click', () => {
+    // Replay
+    function replaySound() {
+        counter = 0;
         music.currentTime = 0;
         video.currentTime = 0;
-    });
+        music.play();
+        video.play();
+        play.src = './assets/img/pause.svg';
 
-    // Change duration 
-    timeSelectors.forEach(selector => {
-        selector.addEventListener('click', () => {
-            maxDuration = selector.getAttribute("data-time");  
-            music.currentTime = 0;
-            video.currentTime = 0;
-            timer.textContent = `${Math.floor(maxDuration / 60)}:${Math.floor(maxDuration % 60)}`;
-        });
-    });
+    };
+
+    function setDuration(id) {
+        let mins = +minutesArea.textContent;
```

Comment:
it is better to use parseInt or parseFloat for better visibility of string to number coersion

Reply:
and again why let? you are not changing it anywhwere